modules/ROOT/pages/_partials/dataweave-formats-flatfile-config.adoc

```diff
@@ -42,6 +42,8 @@ certain single-byte character encodings
 | `segmentIdent` | `String` | `null` | Segment identifier in the schema for fixed width or copybook schemas (only needed when parsing a single segment/record definition and if the schema includes multiple segment definitions).
 | `structureIdent` | `String` | `null` | Structure identifier in schema for flatfile schemas (only needed when parsing a structure definition, and if the schema includes multiple structure definitions)
 | `truncateDependingOn` | `Boolean` | `false` | Truncate COBOL
+| `useMissCharAsDefaultForFill`|`String`| TODO_true_or_false_or |When set to `false`, the flat file reader and writer use spaces for missing characters. When set to `true`, DataWeave uses TODO_WHAT_DOES_IT_DO. _Introduced in DataWeave 4.3 (2.3.0-20210823) for the September 2021 release of Mule 4.3.0-20210823._ See also, the system-level version of this property in xref:dataweave-system-properties.adoc[DataWeave System Properties].
+  TODO: AND DOES SETTING IT ON THE READER OR WRITER OVERRIDE THE SYSTEM SETTING?
```

Comment:
It is only for flat file module (Reader/Writer)

Reply:
```suggestion
```